src/ILVerify/src/ILImporter.StackValue.cs

```diff
@@ -362,7 +362,7 @@ bool IsBinaryComparable(StackValue src, StackValue dst, ILOpcode op)
                         case StackValueKind.ObjRef:
                             return op == ILOpcode.beq || op == ILOpcode.beq_s ||
                                    op == ILOpcode.bne_un || op == ILOpcode.bne_un_s ||
-                                   op == ILOpcode.ceq;
+                                   op == ILOpcode.ceq || op == ILOpcode.cgt_un;
```

Comment:
I agree. It would have been a good idea to mention this instruction in the binary comparison table.